### PR TITLE
Revert changes to server side gRPC codegen

### PR DESCRIPTION
**Pull request by Egorand**

### CODE REVIEW

File: wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt [hunk 1]
```diff
@@ -147,25 +147,29 @@ class KotlinGeneratorTest {
   }
 
   @Test fun blockingSingleRequestSingleResponse() {
+    //language=kotlin
```
Comment (oldergod) [inline wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt]: What are those comments? I don't remember seeing them.

File: wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt [hunk 2]
```diff
@@ -147,25 +147,29 @@ class KotlinGeneratorTest {
...
+           */
+          @WireRpc(
+            path = "/routeguide.RouteGuide/GetFeature",
+            requestAdapter = "routeguide.Point#ADAPTER",
+            responseAdapter = "routeguide.Feature#ADAPTER"
+          )
+          fun GetFeature(request: Point): Feature
+        }
+        
+        """.trimIndent()
```
Comment (oldergod) [inline wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt]: Was the code written without margin before too? Reads nicely. Wonder if that can confuse anyone that this is actually a string and not real code when reading in the middle of it.

File: wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt (hunk 1 above)
Comment (oldergod) [reply wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt]: should we get rid of them?

Comment (Egorand) [reply wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt]: This hints the IDE to add Kotlin highlighting to the multiline string, which makes it nicer to read/edit those fixtures in the IDE.

File: wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt (hunk 2 above)
Comment (Egorand) [reply wire-library/wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt]: This has changed from `trimMargin()` with pipe prefixes to `trimIndent()`. I don't think it's confusing.

**Pull request merged**